.yamato/hdrp_hybrid-osx-metal.yml

```diff
@@ -245,7 +160,7 @@ HDRP_Hybrid_OSX_Metal_Standalone_CUSTOM-REVISION:
       -  ssh -i ~/.ssh/id_rsa_macmini -o "StrictHostKeyChecking=no" bokken@$BOKKEN_DEVICE_IP '$(/usr/local/bin/python3 -m site --user-base)/bin/unity-downloader-cli --source-file ~/Graphics/unity_revision.txt -c editor -c il2cpp  --wait --published-only'
       -  |5-
 
-                 ssh -i ~/.ssh/id_rsa_macmini -o "StrictHostKeyChecking=no" bokken@$BOKKEN_DEVICE_IP "export UPM_REGISTRY=https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates; echo \$UPM_REGISTRY; cd ~/Graphics/TestProjects/HDRP_HybridTests && ~/Graphics/TestProjects/HDRP_HybridTests/utr --suite=playmode --platform=StandaloneOSX --artifacts_path=/Users/bokken/Graphics/TestProjects/HDRP_HybridTests/test-results --testproject=/Users/bokken/Graphics/TestProjects/HDRP_HybridTests --editor-location=/Users/bokken/.Editor --reruncount=2 --compilation-errors-as-warnings --timeout=2400"
+                 ssh -i ~/.ssh/id_rsa_macmini -o "StrictHostKeyChecking=no" bokken@$BOKKEN_DEVICE_IP "export UPM_REGISTRY=https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates; echo \$UPM_REGISTRY; cd ~/Graphics/TestProjects/HDRP_HybridTests && ~/Graphics/TestProjects/HDRP_HybridTests/utr --suite=playmode --platform=StandaloneOSX --artifacts_path=/Users/bokken/Graphics/TestProjects/HDRP_HybridTests/test-results --testproject=/Users/bokken/Graphics/TestProjects/HDRP_HybridTests --editor-location=/Users/bokken/.Editor --extra-editor-arg="-scriptingbackend=mono" --extra-editor-arg="-apicompatibilitylevel=NET_2_0" --extra-editor-arg="-colorspace=Linear"--reruncount=2 --compilation-errors-as-warnings --timeout=2400"
```

Comment:
there is a missing space between `--extra-editor-arg="-colorspace=Linear"--reruncount=2`, probably counts for all metals?

Reply:
ah! there was a missing comma in utr_utils - fixed!